models/Profile/Milestone.js

```diff
@@ -10,6 +10,7 @@ const MilestoneSchema = new Schema({
   date: {
     type: Date,
     required: true,
+    min: '1970-01-01'
```

Comment:
I could see this feature in the official documentation?

Reply:
BTW if you meen in the official documentation, here's the link @eddiejaoude 
https://mongoosejs.com/docs/tutorials/dates.html#validators

Reply:
oh yes, I couldn't see the `min` for dates in the docs, thank you 👍